iso20022-ch-impl/pom.xml

```diff
@@ -58,12 +58,22 @@
             <artifactId>annotations</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+        </dependency>
```

Comment:
Hibernate-validator should not be a required dependency. Instead javax.validation:validation-api should be added as an optional dependency `<optional>true</optional>` (other implementations of the validation-api should be compatible).
hibernate-validator can be changed to `<scope>test</scope>` if required.

Reply:
The tests do not currently cover the bean validation.
As such, replaced hibernate-vlidator with validation-api